app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
 
   validates_presence_of :first_name, :last_name
 
+  def full_name
+    "#{first_name} #{last_name}"
+  end
+
```

Comment:
I agree with this approach, @RobertCram  -- I think that the method does belong in `User`  I think that we should change the code in `MembershipApplicationsHelper  .member_full_name` to now just call this method.